internal/api/migrate/testdata/simple_perses_dashboard.json

```diff
@@ -452,20 +455,23 @@
               "columnSettings": [
                 {
                   "header": "",
-                  "name": "timestamp",
-                  "hide": true
+                  "name": "timestamp"
                 },
                 {
-                  "hide": true,
-                  "name": "__name__"
+                  "header": "ValueCustomHeader",
+                  "name": "value"
                 },
                 {
                   "header": "API server",
                   "name": "apiserver"
                 },
                 {
-                  "header": "ValueCustomHeader",
-                  "name": "value"
+                  "hide": true,
+                  "name": "timestamp"
```

Comment:
weird that the migration is worse here (merged settings getting split), are you sure your maim branch was up to date here?

Reply:
and it doesn't look like I am outdated regarding the table plugin